kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1728,21 +1727,37 @@ Precompiled Contracts
     rule #precompiled(8) => ECPAIRING
     rule #precompiled(9) => BLAKE2F
 
-    syntax Set ::= #precompiledAccounts ( Schedule ) [klabel(#precompiledAccounts), function, total]
+    syntax Int ::= #precompiledAccounts ( Schedule ) [klabel(#precompiledAccounts), function, total]
```

Comment:
`#precompiledAccounts` now returns only the upper bound of the precompiled accounts, assuming they will remain contiguous, starting from 1.

Reply:
> #precompiledAccounts now returns only the upper bound of the precompiled accounts, assuming they will remain contiguous, starting from address 1.
Perhaps it's worth adding this information as an inline comment into the definition?